sdk/resourcemanager/Azure.ResourceManager.Core/src/ApiVersionsBase.cs

```diff
@@ -44,7 +48,11 @@ protected ApiVersionsBase(string value)
         /// <returns></returns>
         public static bool operator >(ApiVersionsBase left, ApiVersionsBase right)
         {
-            if (left is null)
+            if (ReferenceEquals(null, right) && ReferenceEquals(null, left))
```

Comment:
I think we simplify these 3 if statements to
if(left is null)
    return false